### ISSUE TITLE
Chapter 7: splat vs slurp

### ISSUE THREAD

**Issue by 1uc** [chrisvoncsefalvay/learn-julia-the-hard-way#47]
Thanks for the easy to read tutorial.

Chapter 7 gives an example of a 'splat':

    function shout_mi(foods...)

If I compare this to the Julia FAQ [1], I'm not quite sure this is the right way round, it looks more like a slurp. Thanks again for the Tutorial.

[1]: http://docs.julialang.org/en/release-0.4/manual/faq/#what-does-the-operator-do

**Comment by chrisvoncsefalvay** [comment]
Correct! I will amend that chapter in the next few weeks with a lot of new information to bring it up to date to 0.6.0-dev.

Thanks @1uc
